schema/revert/shipping/views.sql

```diff
@@ -2251,56 +2249,38 @@ create or replace view shipping.scan_hcov19_positives_v1 as
     select
         hcov19_result_release_date,
         count(*) filter (where hcov19_result in ('positive', 'inconclusive')) as total_hcov19_positives,
+        count(*) filter (where hcov19_result = 'negative') as total_hcov19_negatives,
         count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county = 'king') as king_county_positives,
+        count(*) filter (where hcov19_result = 'negative' and county = 'king') as king_county_negatives,
         count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county = 'snohomish') as snohomish_county_positives,
+        count(*) filter (where hcov19_result = 'negative' and county = 'snohomish') as snohomish_county_negatives,
         count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county = 'yakima') as yakima_county_positives,
-        count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county is null) as other_positives
+        count(*) filter (where hcov19_result = 'negative' and county = 'yakima') as yakima_county_negatives,
+        count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county is null) as other_positives,
+        count(*) filter (where hcov19_result = 'negative' and county is null) as other_negatives
     from scan_hcov19_results
     group by hcov19_result_release_date
 ;
 
-comment on view shipping.scan_hcov19_positives_v1 is
-  'A view of counts of hcov19 positives from the SCAN project grouped by date results were released.';
+comment on view shipping.scan_hcov19_result_counts_v1 is
+  'A view of counts of hcov19 results from the SCAN project grouped by date results were released.';
 
--- Even if it's just aggregate counts of hcov19 positives,
+-- Even if it's just aggregate counts of hcov19 results,
 -- we should probably restrict access to this view to only hcov19-visibility
 -- and scan-dashboard-exporter.
 --  -Jover, 9 July 2020
-revoke all on shipping.scan_return_results_v1 from reporter;
```

Comment:
How is the scan_result_results_v1 view related to this PR? How is reporter related to this PR?

Reply:
It's not related to this PR at all. This revert script is automatically generated by sqitch. It basically copies the deploy script over, so when we run `sqitch revert` it will revert back to the previous state. 
The only actual changes I made to this revert script is dropping lines 19-20.

Reply:
GitHub doesn't show this sort of change well in its diffs, but on the command-line, git can use rename/copy detection to show a much nicer diff when a file is renamed/copied with some changes.